gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/CommonSteps.cs

```diff
@@ -131,9 +132,16 @@ public void TranslateTraversal(string traversalText)
             {
                 _g = _g.WithComputer();
             }
-            
-            _traversal =
-                Gremlin.UseTraversal(ScenarioData.CurrentScenario.Name, _g, _parameters);
+
+            try
+            {
+                _traversal =
+                    Gremlin.UseTraversal(ScenarioData.CurrentScenario.Name, _g, _parameters);
+            }
+            catch (Exception ex)
+            {
+                _error = ex;
```

Comment:
could you say what exception is expected here? i think `UseTraversal()` just gets grabs the `Traversal` object. it doesn't iterate it. there shouldn't be any errors on construction that i can think of. am i forgetting something?

Reply:
i'd be in favor of removing that for the non-JVM stuff. it definitely would matter for Java where you could trigger a problem during the building of the traversal in the parse mechanism, but I think i'd rather have a reason for that happening in the non-JVM parsing.

Reply:
So remove this code then?

Reply:
yes. let's make sure it works without it.

Reply:
done